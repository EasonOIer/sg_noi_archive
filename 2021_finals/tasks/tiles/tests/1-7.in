8 8
x.xxxxxx
x....xx.
...xxx.x
2 1 8
1 1 5
1 2 8
2 2 8
2 2 8
2 1 8
2 2 8
1 2 5
