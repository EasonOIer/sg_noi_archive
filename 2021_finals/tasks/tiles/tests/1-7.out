37
40
40
59
40
